packages/bitcore-node/src/logger.ts

```diff
@@ -11,4 +11,29 @@ const logger = new winston.Logger({
   ]
 });
 
+const timezone = new Date()
+  .toLocaleString('en-US', { timeZoneName: 'short' })
+  .split(' ')
+  .pop();
+
+export const formatTimestamp = (date: Date) =>
+  `${date.getFullYear()}-${(date.getMonth() + 1).toString().padStart(2, '0')}-${date
+    .getDate()
+    .toString()
+    .padStart(2, '0')} ${date
+    .getHours()
+    .toString()
+    .padStart(2, '0')}:${date
+    .getMinutes()
+    .toString()
+    .padStart(2, '0')}:${date
+    .getSeconds()
+    .toString()
+    .padStart(2, '0')}.${date
+    .getMilliseconds()
+    .toString()
+    .padEnd(3, '0')} ${timezone}`;
+
+export const timestamp = () => formatTimestamp(new Date());
+
 export default logger;
```

Comment:
Can this line be deleted given:
https://github.com/bitpay/bitcore/pull/2145/files#diff-87388ecdc1725fb9c6acc98785273ceeR5

Reply:
Yep! Deleted and reverted the import to using the default import